app/screens/channel_info/options/index.tsx

```diff
@@ -32,6 +39,9 @@ const Options = ({channelId, type, callsEnabled}: Props) => {
             {type !== General.DM_CHANNEL &&
                 <Members channelId={channelId}/>
             }
+            {canManageMembers &&
```

Comment:
is the observer for this taking into account Direct Messages and town square? if so ignore my previous comment, if not, then we need the check here

Reply:
`observeCanManageChannelMembers` checks for:
- Channel exist
- Channel is not deleted
- Channel is not DM or GM
- Channel name is not the default channel name (aka town square)
- If the channel is public, the user has `MANAGE_PUBLIC_CHANNEL_MEMBERS` permissions for that channel
- If the channel is private, the user has `MANAGE_PRIVATE_CHANNEL_MEMBERS` permissions for that channel

That should cover all the cases for this, but let me know if we are missing any.

Reply:
awesome